### ISSUE TITLE
Create a Node with hyperlink of local file or folder if drag and drop

### ISSUE THREAD

**Issue by VarinderMick** [juzzlin/Heimer#256]
Add feature: to Create Nodes with hyperlink of local file or folder if drag and drop on MAP area or on create branches of Node on which the files or folders dropped. node text shows only Folder/file NAME not full path but with hyperlink may be inside some icon or text hidden only shown while need to edit or change Otherwise full path will spoil the node beauty/visuals.
love the simple and fast concept of your work. Thanks!

**Comment by juzzlin** [comment]
Yes, I'm currently working on some special type of nodes. Let's see how that will work out :)

**Comment by gleyconJ** [comment]
Hello my programmer friend! I would like to join this conversation to say that I am another person who would really like this feature to come to light. I have been a mind map user for over 10 years. And I have hundreds of them created, and since I first saw your program I've been watching it to see when it can be part of my workflow. As it is, I haven't been able to use it professionally yet (only in ideas for games and tests). However, I see huge potential for it to be something truly functional and take me out of the world of paid apps.

VarinderMick was clear and killer in a resource that will, exponentially, greatly increase the implementation of this program in our daily lives.

I believe that the best program to give you inspiration for this resource in question would be this one:

[](https://github.com/KnowledgeCanvas/knowledge)

Could you take a look at the "graph" implementation of this program and see how it works with external files.

I am a QT user. And I'm trying to leave all my applications in this language. Heimer would replace Knoledge if you implemented this.

Thank you very much in advance.
